Vagrantfile

```diff
@@ -165,26 +151,21 @@ Vagrant.configure(2) do |config|
     ${HOME}/bin/direnv allow ${HOME}/scf
   SHELL
 
-  config.vm.provision "shell", privileged: false, inline: <<-SHELL
-    set -e
-
+  # Install common and dev tools
+  config.vm.provision :shell, privileged: true, inline: <<-SHELL
+    set -o errexit -o xtrace -v
     # Get proxy configuration here
-    source /etc/environment
-    export no_proxy http_proxy https_proxy NO_PROXY HTTP_PROXY HTTPS_PROXY
-
-    # Install development tools
-    (
-      cd "${HOME}/scf"
-      ${HOME}/bin/direnv exec ${HOME}/scf/bin/dev/install_tools.sh
-    )
-
+    export HOME=/home/vagrant
+    export PATH=$PATH:/home/vagrant/bin
+    cd "${HOME}/scf"
+    bash ${HOME}/scf/bin/common/install_tools.sh
```

Comment:
If I'm reading things right, both `SCF_BIN_DIR` and `SCF_TOOLS_DIR` are unset at this point; this means they default to the locations in `/home/vagrant`.  And then we proceed to drop a bunch of files there as root.  That is Not Cool — files in the home directory shouldn't be owned by random other people (in this case, root).

```
vagrant@vagrant-kube:~> ls -laR bin tools
bin:
total 3188
drwxr-xr-x  2 vagrant users    4096 Jul 27 18:35 .
drwxr-xr-x 12 vagrant users    4096 Jul 27 18:54 ..
-rwxr-xr-x  1 vagrant users 3255774 May 25 08:14 direnv

tools:
total 6400
drwxr-xr-x  2 root    root     4096 Jul 27 18:35 .
drwxr-xr-x 12 vagrant users    4096 Jul 27 18:54 ..
-rw-r--r--  1 root    root  6543090 Sep 13  2016 cf.tgz

vagrant@vagrant-kube:~> ls -l /usr/local/bin/
total 161972
-rwxr-xr-x 1 root    root     4650644 Jul 26 23:17 certstrap
-rwxr-xr-x 1 root    root    23610336 Aug 10  2016 cf
-rwxrwxr-x 1 vagrant vagrant 63247746 May 17 19:33 helm
-rwxr-xr-x 1 root    root    69968469 Jul 27 18:38 helm-certgen
-rwxr-xr-x 1 root    root     2405672 Jun  9 14:31 k
-rwxr-xr-x 1 root    root        4224 Jul 27 18:35 kk
-rwxr-xr-x 1 root    root     1951738 May 23 17:58 stampy
```
So it looks like `bin/` ends up being owned by `vagrant` (good), but `tools` is owned by `root` (bad).  Also, the bin dir (except for direnv‽) is `/usr/local/bin`, somehow.  And not sure why `vagrant` owns `helm`…